.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -15,10 +15,8 @@ about: Create a report to help us improve
 
 **What is the current behavior?**
 
-
```

Comment:
The spacing is intentional bigger than normal, as the user want to start writing here. Maybe exclude the issue/pr templates from prettier.